packages/core/admin/admin/src/components/Filters.tsx

```
@@ -481,37 +603,44 @@ const AttributeTag = ({
+  let formattedValue: string = decodedValue;
+
+  if (!FILTERS_WITH_NO_VALUE.includes(operator)) {
+    switch (type) {
+      case 'date':
+        formattedValue = formatDate(decodedValue, { dateStyle: 'full' });
+        break;
+      case 'datetime':
+        formattedValue = formatDate(decodedValue, { dateStyle: 'full', timeStyle: 'short' });
+        break;
+      case 'time':
+        const [hour, minute] = decodedValue.split(':');
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)